tiled/client/base.py

```diff
@@ -192,6 +196,12 @@ def metadata(self):
         # persistent.
         return DictView(self._item["attributes"]["metadata"])
 
+    def metadata_copy(self):
```

Comment:
While we consider whether or not to keep `DictView` I'd suggest we hold off on adding this just yet. In the mean time we can recommend `copy.deepcopy(self.metadata)` in docstrings.

If we decide to keep `DictView`, we can add this in a follow-up PR. But once added it's harder to remove it.

Reply:
Thinking about this more, and speaking with @Kezzsim, I'm appreciating that there isn't really a way around this. I've come around on keeping this method.